public/survey.html

```diff
@@ -0,0 +1,157 @@
+				    <option value="1">1 (Strongly agree)</option>
+				    <option value="2">2</option>
+				    <option value="3">3</option>
+				    <option value="4">4</option>
+				    <option value="5">5 (Strongly disagree)</option>
+				</select>
+				<h3>Question 5</h3>
+				<h4>You hate sleeping for 8 or more hours. </h4>
+				<select data-placeholder="" class="chosen-select" id="q5" required>
+				    <option value="">Seclect an option</option>
+				    <option value="1">1 (Strongly agree)</option>
+				    <option value="2">2</option>
+				    <option value="3">3</option>
+				    <option value="4">4</option>
+				    <option value="5">5 (Strongly disagree)</option>
+				</select>
+			</form>	
+		</div>
+		<br>
+		<div class="row">
+			<button class="btn btn-lg btn-primary" id="submit" data-toggle="modal" data-target="#myModal">Submit</button>
+			<!-- Modal -->
+			<div id="myModal" class="modal fade" role="dialog">
+			  <div class="modal-dialog">
+
+			    <!-- Modal content-->
+			    <div class="modal-content">
+			      <div class="modal-header">
+			       	<button type="button" class="close" data-dismiss="modal">&times;</button>
+			        <h1 class="modal-title">Best Match</h1>
+			      </div>
+			      <div class="modal-body">
+			        <h3><p id="modal-name"></p></h3>
+			        <p id="modal-photo"></p>
+			      </div>
+			      <div class="modal-footer">
+			        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+			      </div>
+			    </div>
+			  </div>
+			</div> 
+		</div>
+		<hr>
+		<div>
+			<a href="/api/friends" target="blank">API FRiends List</a> | <a href="https://github.com/nchandika1/FriendFinder" target="blank">Git Hub Repo</a>
+		</div>
+	</div>
+<script type="text/javascript">
+	function clearForm() {
+		$("#name").val('');
+		$("#photo").val('');
+		$("#q1").val('');
+		$("#q2").val('');
+		$("#q3").val('');
+		$("#q4").val('');
+		$("#q5").val('');
+	}
+
+	$("#submit").on("click", function() {
+		// Make sure the form fields are filled completely
+		if ( ($("#name").val().trim() === "") ||
+			($("#photo").val().trim() === "") ||
+			($("#q1").val() === "") ||
+			($("#q2").val() === "") ||
+			($("#q3").val() === "") ||
+			($("#q4").val() === "") ||
+			($("#q5").val() === "") ) {
+				alert("Please fill out all the fields.");
+				$("#test").hide();
+				return;		
+		}
+
+		// Set up the data structure to send to the server
+		var userData = {
+	    		name: $("#name").val(),
+	    		photo: $("#photo").val(),
+	    		scores: [$("#q1").val(), $("#q2").val(), $("#q3").val(), $("#q4").val(), $("#q5").val()]
+	    	}
+		
+		// Send POST AJAC call to the server
+		$.post("/new", userData)
+		.then(function(data) {
+			// Display modal with the best match sent by the server
+			$("#modal-name").text(data.name);
+			$("#modal-photo").html("<img src=" + data.photo + " width=200 height=200>");
+
+			// Let us clear the form now
+			clearForm();
+		});
+	});
+</script>
```

Comment:
It is good that you know how to perform these functions using the jquery library, however, I would challenge you to do it without the assistance of jquery. In this instance, you could surround your input fields in a form and add action and method to the form and as long as you have body-parser then your post route will receive that data on submission.